src/components/layout/StyledRowLeft.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledRowLeftSide = styled.div`
+color: yellow;
+width: 100%;
+height: 170px;
+
+/* border: 2px solid black; */
```

Comment:
- [ ] Since we are preparing this code to be production-code, we should delete all commented code.